CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing
+
+Thanks for your interest in contributing to tslint-microsoft-contrib!
```

Comment:
Maybe it make sense to add backtics around tslint-microsoft-contrib as in next sentence?